string color_name
---
bool success
string message
